### ISSUE TITLE
Usage with Lumen

### ISSUE THREAD

**Issue by coolycow** [matchory/elasticsearch#47]
Hi!

The description indicates that the package supports Lumen. 

But apparently there is some kind of incompatibility:
```
local.ERROR: Matchory\Elasticsearch\ElasticsearchServiceProvider::Matchory\Elasticsearch\{closure}(): Argument #1 ($app) must be of type Illuminate\Contracts\Foundation\Application, Laravel\Lumen\Application given, called in /var/www/myproject/vendor/illuminate/container/Container.php on line 832 {"exception":"[object] (TypeError(code: 0): Matchory\\Elasticsearch\\ElasticsearchServiceProvider::Matchory\\Elasticsearch\\{closure}(): Argument #1 ($app) must be of type Illuminate\\Contracts\\Foundation\\Application, Laravel\\Lumen\\Application given, called in /var/www/myproject/vendor/illuminate/container/Container.php on line 832 at /var/www/myproject/vendor/matchory/elasticsearch/src/ElasticsearchServiceProvider.php:193)
[stacktrace]
#0 /var/www/myproject/vendor/illuminate/container/Container.php(832): Matchory\\Elasticsearch\\ElasticsearchServiceProvider->Matchory\\Elasticsearch\\{closure}()
#1 /var/www/myproject/vendor/illuminate/container/Container.php(717): Illuminate\\Container\\Container->build()
#2 /var/www/myproject/vendor/illuminate/container/Container.php(655): Illuminate\\Container\\Container->resolve()
#3 /var/www/myproject/vendor/laravel/lumen-framework/src/Application.php(300): Illuminate\\Container\\Container->make()
#4 /var/www/myproject/vendor/matchory/elasticsearch/src/ElasticsearchServiceProvider.php(52): Laravel\\Lumen\\Application->make()
#5 /var/www/myproject/vendor/illuminate/container/BoundMethod.php(36): Matchory\\Elasticsearch\\ElasticsearchServiceProvider->boot()
#6 /var/www/myproject/vendor/illuminate/container/Util.php(40): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#7 /var/www/myproject/vendor/illuminate/container/BoundMethod.php(93): Illuminate\\Container\\Util::unwrapIfClosure()
#8 /var/www/myproject/vendor/illuminate/container/BoundMethod.php(37): Illuminate\\Container\\BoundMethod::callBoundMethod()
#9 /var/www/myproject/vendor/illuminate/container/Container.php(614): Illuminate\\Container\\BoundMethod::call()
#10 /var/www/myproject/vendor/laravel/lumen-framework/src/Application.php(277): Illuminate\\Container\\Container->call()
#11 /var/www/myproject/vendor/laravel/lumen-framework/src/Application.php(262): Laravel\\Lumen\\Application->bootProvider()
#12 /var/www/myproject/vendor/laravel/lumen-framework/src/Console/Kernel.php(114): Laravel\\Lumen\\Application->boot()
#13 /var/www/myproject/artisan(35): Laravel\\Lumen\\Console\\Kernel->handle()
#14 {main}
"} 
```

**Comment by Radiergummi** [comment]
Hey there, yup, that's a regression. Going to fix this on monday! ✌️

**Comment by Radiergummi** [comment]
Fixed in e35dd8c

**Comment by Radiergummi** [comment]
@coolycow released as 2.5.2. Let me know if you hit any other roadblocks with Lumen. We're not using it in our stack, but I'd like to commit to keeping Lumen support in.

**Comment by coolycow** [comment]
@Radiergummi Hey!
All perfectly! Thank you very much for your help.
